### ISSUE TITLE
docs-build fails 

### ISSUE THREAD

**Issue by registerrier** [gammapy/gammapy#4396]
docs-build is failing both in CI and locally. 
The issue comes from ipywidgets that cannot find ipykernel. 
See e.g. https://github.com/gammapy/gammapy/actions/runs/4481165613/jobs/7877487789?pr=4352#step:6:1189

**Comment by registerrier** [comment]
The issue is connected with the latest version of ipywidget, that was released on March 21st .

Currently, if one installs `pip install gammapy[all]` and does `import ipywidget`, the result is:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[1], line 1
----> 1 import ipywidgets

File ~/Code/mambaforge/envs/test_widgets/lib/python3.9/site-packages/ipywidgets/__init__.py:59
     56         return
     57     register_comm_target()
---> 59 _handle_ipython()

File ~/Code/mambaforge/envs/test_widgets/lib/python3.9/site-packages/ipywidgets/__init__.py:57, in _handle_ipython()
     55 if ip is None:
     56     return
---> 57 register_comm_target()

File ~/Code/mambaforge/envs/test_widgets/lib/python3.9/site-packages/ipywidgets/__init__.py:47, in register_comm_target(kernel)
     45 def register_comm_target(kernel=None):
     46     """Register the jupyter.widget comm target"""
---> 47     comm_manager = get_comm_manager()
     49     comm_manager.register_target('jupyter.widget', Widget.handle_comm_opened)
     50     comm_manager.register_target('jupyter.widget.control', Widget.handle_control_comm_opened)

File ~/Code/mambaforge/envs/test_widgets/lib/python3.9/site-packages/ipywidgets/__init__.py:33, in get_comm_manager()
     30 def get_comm_manager():
     31     ip = get_ipython()
---> 33     if ip is not None and ip.kernel is not None:
     34         return get_ipython().kernel.comm_manager

AttributeError: 'TerminalInteractiveShell' object has no attribute 'kernel'
```
This is clearly an issue on the side of ipywigets. The temporary fix is to force it at 8.0.4.

**Comment by registerrier** [comment]
Solved by #5232
